tests/regression/task_spec.py

```diff
@@ -34,11 +37,19 @@ class TaskSpec:
     Task specification representing testing scenario.
     Contains a list of functions to be compared with DiffKemp during the test.
     """
-    def __init__(self, spec, task_name, tasks_path, kernel_path):
+    def __init__(self, spec, task_name, tasks_path, kernel_path,
+                 patterns_path):
```

Comment:
It would be nice to have a default `patterns_path` here so that you don't have to set it from each `TaskSpec` extending class.

Reply:
Removing related `TaskSpec` attributes in order to resolve this.